src/components/atoms/CardContent/CardContent.css.ts

```diff
@@ -0,0 +1,48 @@
+import { style } from "@vanilla-extract/css";
+
+import { sprinkles } from "@/styles/sprinkles.css";
+
+export const box = style([
+  sprinkles({
+    display: "flex",
+    flexDirection: "column",
+    gap: "lg",
+  }),
+  {
+    alignItems: "flex-start",
+    flex: "1 0 0",
+  },
+]);
+
+export const content = style([
+  sprinkles({
+    display: "flex",
+    alignItems: "center",
+    gap: "sm",
+  }),
+]);
```

Comment:
`sprinkles`만 사용할 때에는 `style` 사용하지 않아도 됩니다

```suggestion
export const content = sprinkles({
  display: "flex",
  alignItems: "center",
  gap: "sm",
});
```

Reply:
[a41fd74](https://github.com/sparcs-kaist/ara-web/pull/13/commits/a41fd74748ecfd604052c87d590b44233f74efcf)